src/main/java/io/jenkins/plugins/ml/IPythonKernelInterpreter.java

```diff
@@ -0,0 +1,21 @@
+package io.jenkins.plugins.ml;
+
+public class IPythonKernelInterpreter implements KernelInterpreter {
+
+    /**
+     * IPython will be connected and interpreted through the object of this class.
+     * @param code - python code to be executed
+     * @return null
+     */
+    @Override
+    public String interpretCode(String code) {
+        // TODO implemet the interpret function
```

Comment:
nit-pick, feel free to ignore, but `s/implemet/implement`

Reply:
yup thanks